_topic_map.yml

```diff
@@ -885,6 +885,19 @@ Topics:
 - Name: Administrator CLI commands
   File: administrator-cli-commands
   Distros: openshift-enterprise,openshift-origin
+- Name: OpenShift Developer CLI
```

Comment:
@cdrage How are people referring to this CLI? Do people actually ever say "OpenShift Do", or do they just say odo? Because in an above comment I suggested putting "(odo)" at the end, because that's what is used to refer to it all throughout the rest of this PR, and if odo is how people are going to refer to it, then that's probably what should be there instead of "OpenShift Do". wdyt?

If it helps, for oc, as a docs team we decided that calling it "OpenShift CLI (`oc`)" was the preferred way to refer to it. And I was probably going to make a subfolder with this name for the existing CLI docs.

If you think that "OpenShift Do" is important, we could say "OpenShift Do (odo)", or maybe "OpenShift Do developer CLI (odo)" to make it clearer it's for developers.

Reply:
They refer to it as just `odo` :) 

Although it would be nice to have one place were we refer to it as OpenShift Do so people at least know what the acronym is about.

`OpenShift Do developer CLI (odo)` sounds perfect actually. Explains the acronym as well that it's a developer CLI.